symphony/content/content.systemauthors.php

```diff
@@ -566,6 +570,17 @@ public function __actionNew()
         if (@array_key_exists('save', $_POST['action']) || @array_key_exists('done', $_POST['action'])) {
             $fields = $_POST['fields'];
 
+            $canCreate = Symphony::Author()->isDeveloper() || Symphony::Author()->isManager();
+
+            if (!$canCreate) {
+                $this->pageAlert(__('You don\'t have the rights to create a new author.'), Alert::ERROR);
+                return;
+            }
```

Comment:
We can get rid of the page alert since we block this already

Reply:
`we block this already`
Nope. This is where the validation is made.

Reply:
See https://github.com/symphonycms/symphony-2/pull/2815/files#diff-9375ed02f5d8084262cfb796bef2eb9fR218 (Line 218). The use should never reached those lines. The problem is that actions are processed before the views: This is why it is needed to valide access when changing the value.

Reply:
As discussed we will keep the validation in the post and I will change the page alert for a `throwCustomError` That way we will have the validation on the GET request and the POST

Reply:
:+1: